thousandeyes-sdk-core/test/test_api_client.py

```diff
@@ -0,0 +1,124 @@
+import pytest
+import datetime
+from unittest.mock import Mock, patch
+from thousandeyes_sdk.core.api_client import ApiClient
+from thousandeyes_sdk.core.configuration import Configuration
+from thousandeyes_sdk.core import rest
+from thousandeyes_sdk.core.exceptions import ApiException
+
+
+@pytest.fixture
+def api_client():
+    config = Configuration()
+    return ApiClient(configuration=config)
+
+
+def test_api_client_initialization(api_client):
+    assert api_client.configuration is not None
+    assert isinstance(api_client.rest_client, rest.RESTClientObject)
+    assert api_client.default_headers == {}
+    assert api_client.cookie is None
```

Comment:
Might be best to wait for [these changes](https://github.com/thousandeyes/thousandeyes-sdk-python/pull/56/files ) since they also ended up affecting the api client. Not sure yet to what extent that will impact these tests (also dependeds on whether we decide to remove more or less stuff) but `cookie`, for example

Reply:
yep, makes sense. Will wait